tools/lsp/util.rs

```diff
@@ -44,15 +47,16 @@ pub fn map_range(sf: &SourceFile, range: TextRange) -> lsp_types::Range {
     lsp_types::Range::new(map_position(sf, range.start()), map_position(sf, range.end()))
 }
 
-pub fn map_to_offset(sf: &SourceFile, position: lsp_types::Position) -> usize {
-    sf.offset(
+pub fn map_text_size(sf: &SourceFile, position: lsp_types::Position) -> TextSize {
```

Comment:
I suggest to keep the name `map_to_offset`, it was more explicit and `TextSize` is representing an offset.
at least it is not "mapping the text size", so the new name isn't accurate.

Or maybe `position_to_text_size`?

Reply:
I think you will like the new names better.